i18n/it/cosmic_settings.ftl

```diff
@@ -241,195 +245,195 @@ x-hours = { $number ->
 ## Desktop: Workspaces
 
 workspaces = Spazi di lavoro
-    .desc = Imposta numero, comportamento e posizione degli spazi di lavoro.
+    .desc = Imposta numero spazi di lavoro, il loro comportamento e la loro posizione.
 
 workspaces-behavior = Comportamento spazi di lavoro
     .dynamic = Spazi di lavoro dinamici
-    .dynamic-desc = Rimuovi automaticamente spazi di lavoro vuoti.
-    .fixed = Numero fisso di spazi di lavoro
-    .fixed-desc = Aggiungi o rimuovi spazi di lavoro nell'anteprima.
+    .dynamic-desc = Rimuovi automaticamente gli spazi di lavoro vuoti
+    .fixed = Spazi di lavoro limitati
+    .fixed-desc = Aggiungi o rimuovi gli spazi di lavoro
 
-workspaces-multi-behavior = Comportamento con più schermi
-    .span = Gli spazi di lavoro comprendono più schermi
-    .separate = Ogni schermo ha il suo spazio di lavoro
+workspaces-multi-behavior = Comportamento multi-schermo
+    .span = Gli spazi di lavoro si espandono in tutti gli schermi
+    .separate = Ogni schermo ha gli spazi di lavoro separati
 
-workspaces-overview-thumbnails = Anteprime Spazio di Lavoro
-    .show-number = Mostra Numero Spazio di Lavoro
-    .show-name = Mostra Nome Spazio di Lavoro
+workspaces-overview-thumbnails = Anteprima spazi di lavoro
+    .show-number = Mostra il numero dello spazio di lavoro
+    .show-name = Mostra il nome dello spazio di lavoro
 
-workspaces-orientation = Orientamento Spazi di Lavoro
+workspaces-orientation = Orientamento spazi di lavoro
     .vertical = Verticale
     .horizontal = Orizzontale
 
 ## Networking: Wired
 
-wired = Cablata
-    .desc = Connessione cablata, profili di connessione
+wired = Via cavo
+    .desc = Connessione via cavo, profili di connessione
 
 ## Networking: Online Accounts
 
-online-accounts = Account Online
-    .desc = Aggiungi account, IMAP e SMTP, login aziendali
+online-accounts = Account online
+    .desc = Aggiungi account IMAP, SMTP e enterprise
 
 ## Time & Language
 
-time = Ora e Lingua
+time = Orario e linguaggio
     .desc = N/A
 
 time-date = Data e ora
-    .desc = Fuso orario, impostazioni orologio automatiche, e alcuni formati ora.
+    .desc = Fuso orario, impostazioni orologio automatico e formattazione orario
```

Comment:
here "automatico" refers to "impostazioni", so it should be feminine plural, like it was.

Reply:
Hmm, you're right.
Thanks!